media/js/test/spec/send-to-device.js

```diff
@@ -0,0 +1,192 @@
+/* Base JS unit test spec for bedrock send-to-device.js
+ * For reference read the Jasmine and Sinon docs
+ * Jasmine docs: http://pivotal.github.io/jasmine/
+ * Sinon docs: http://sinonjs.org/docs/
+ */
+
+describe('send-to-device.js', function() {
+
+    var form;
+    var spinnerStub;
+
+    beforeEach(function () {
+
+        var formMarkup = [
+            '<section id="send-to-device">' +
+                '<div class="form-container">' +
+                    '<form id="send-to-device-form">' +
+                        '<ul class="error-list hidden">' +
+                            '<li>An error occurred in our system. Please try again later.</li>' +
+                        '</ul>' +
+                        '<div class="input">' +
+                            '<input type="hidden" id="id-platform" value="all">' +
+                            '<label id="form-input-label" for="id-input" data-alt="Enter your email or 10-digit phone number.">Enter your email.</label>' +
+                            '<div class="inline-field">' +
+                                '<input id="id-input" type="text" data-alt="Enter your email or 10-digit phone number." placeholder="Enter your email." required>' +
+                                '<button type="submit" class="button-flat">Send</button>' +
+                            '</div>' +
+                        '</div>' +
+                        '<div class="thank-you hidden">' +
+                            '<p>Check your device for the email or text message!</p>' +
+                            '<a href="#" role="button" class="more send-another">Send to another device</a>' +
+                        '</div>' +
+                        '<div class="loading-spinner"></div>' +
+                        '</form>' +
+                    '</div>' +
+                '<footer>' +
+                    '<ul>' +
+                        '<li class="app-store">' +
+                            '<a class="more" href="#">Go to the App Store</a>' +
+                        '</li>' +
+                        '<li class="google-play">' +
+                            '<a class="more" href="#">Go to Google Play</a>' +
+                        '</li>' +
+                    '</ul>' +
+                '</footer>' +
+            '</section>'
+        ].join();
```

Comment:
Man, it sure would be nice to not have to write markup like this for tests. :persevere: